libmscore/edit.cpp

```diff
@@ -3544,7 +3544,7 @@ void Score::undoChangeClef(Staff* ostaff, Element* e, ClefType ct)
 
       Clef* gclef = 0;
       Fraction tick = e->tick();
-      Fraction rtick = e->rtick();
+      Fraction rtick = e->isMeasure() ? Fraction::fromTicks(0) : e->rtick();
```

Comment:
There is no reason to construct a Fraction from ticks here. `Fraction(0, 1)` is preferred.

Also, Measure::rtick() now currently returns Fraction(0, 1) anyway. This would simply be a guard against failure if that were to change in the future.

Reply:
Agreed. I'll remove this actual fix, but I'll leave the mtest. It's up to Anatoly whether it should be merged or not.